<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTokenDialog</class>
 <widget class="QWidget" name="CreateTokenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Token</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="frameTokenData">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout1">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>The name of the new tokens to create (Required)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="addressLabel">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QValidatedLineEdit" name="addressLineEdit">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="placeholderText">
             <string>The address to send from (Required)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="categoryLabel">
            <property name="text">
             <string>Category:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="categoryLineEdit">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>A category for the new tokens (Optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="subcategoryLabel">
            <property name="text">
             <string>Subcategory:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="subcategoryLineEdit">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>A subcategory for the new tokens (Optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="urlLabel">
            <property name="text">
             <string>URL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="urlLineEdit">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>A URL for further information about the new tokens (Optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="dataLabel">
            <property name="text">
             <string>Data:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dataLineEdit">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="placeholderText">
             <string>A description for the new tokens (Optional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="ecosystemLabel">
            <property name="text">
             <string>Ecosystem:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ecosystemComboBox">
            <property name="toolTip">
             <string>The ecosystem to create the tokens in.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="quantityLabel">
            <property name="text">
             <string>Amount:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="quantityLineEdit">
            <property name="minimumSize">
             <size>
              <width>133</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>240</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Amount of tokens to create.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>(Required if not managed)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="emptyLabel2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="divisibleCheckBox">
            <property name="toolTip">
             <string>Make tokens divisible (8 decimal places)</string>
            </property>
            <property name="text">
             <string>Divisible</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="emptyLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="managedCheckBox">
            <property name="toolTip">
             <string>If the owner of this token will be able to issue more tokens in the future</string>
            </property>
            <property name="text">
             <string>Managed</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>1000</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Confirm the create token action.</string>
       </property>
       <property name="text">
        <string>C&amp;reate</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear all fields of the form.</string>
       </property>
       <property name="text">
        <string>Clear &amp;All</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
